### PR TITLE
BZ2229164: Minor update to bridge NAD YAML

### PR DESCRIPTION
**Pull request by sjhala-ccs**
Version(s): 4.12+

Issue: https://bugzilla.redhat.com/show_bug.cgi?id=2229164

Link to docs preview: https://63552--docspreview.netlify.app/openshift-enterprise/latest/virt/vm_networking/virt-connecting-vm-to-linux-bridge#virt-creating-linux-bridge-nad-cli_virt-connecting-vm-to-linux-bridge

QE review:
- [x] QE has approved this change.

### CODE REVIEW

**Comment by yossisegev** [comment]
/lgtm
(CNV network QE)

File: modules/virt-creating-linux-bridge-nad-cli.adoc [hunk 1]
```diff
@@ -48,6 +49,7 @@ spec:
 <5> The name of the Linux bridge configured on the node.
 <6> Optional: Flag to enable MAC spoof check. When set to `true`, you cannot change the MAC address of the pod or guest interface. This attribute provides security against a MAC spoofing attack by allowing only a single MAC address to exit the pod.
 <7> Optional: The VLAN tag. No additional VLAN configuration is required on the node network configuration policy.
+<8> Indicates whether the default VLAN must be preserved on the virtual ethernet (veth) end connected to the bridge. This must be set to `false`.
```
Comment (phoracek) [inline modules/virt-creating-linux-bridge-nad-cli.adoc:52]:
```suggestion
<8> Indicates whether the VM will be connected to the bridge through the default VLAN.
```

File: modules/virt-creating-linux-bridge-nad-cli.adoc [hunk 2]
```diff
@@ -38,7 +38,8 @@ spec:
     "type": cnv-bridge, <4>
     "bridge": bridge-interface, <5>
     "macspoofchk": true, <6>
-    "vlan": 1 <7>
+    "vlan": 1, <7>
```
[the remark this answers is not in the document]
Comment (phoracek) [reply modules/virt-creating-linux-bridge-nad-cli.adoc:41]: We should make it something else than 1 (which is typically the default VLAN)

**Review by phoracek — CHANGES_REQUESTED** [review]

**Comment by sjhala-ccs** [comment]
@phoracek I incorporated your comments. Can you please review again?

**Comment by stevsmit** [comment]
lgtm

**Review by phoracek — APPROVED** [review]

**Pull request merged**